promtail-config-prod.yml

```
@@ -0,0 +1,37 @@
+          job: varlogs
+          env: prod
+          __path__: /log/warn/**/*.log
+
+  - job_name: spring-logs-error
+    static_configs:
+      - targets:
+          - localhost
+        labels:
+          job: varlogs
+          env: prod
+          __path__: /log/error/**/*.log
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)